File: examples/museum/oseg_generated/php/BuyMuseumTickets_event_entry.php
<?php

require_once __DIR__ . '/vendor/autoload.php';

$config = OpenAPIMuseum\Client\Configuration::getDefaultConfiguration();

$buy_museum_tickets = (new OpenAPIMuseum\Client\Model\BuyMuseumTickets())
    ->setTicketType(OpenAPIMuseum\Client\Model\BuyMuseumTickets::TICKETTYPE_EVENT)
    ->setTicketDate(new \DateTime("2023-09-05"))
    ->setEmail("todd@example.com")
    ->setTicketId(null)
    ->setEventId("dad4bce8-f5cb-4078-a211-995864315e39");

try {
    $response = (new OpenAPIMuseum\Client\Api\TicketsApi($config))->buyMuseumTickets(
        buy_museum_tickets: $buy_museum_tickets,
    );

    print_r($response);
} catch (OpenAPIMuseum\Client\ApiException $e) {
    echo 'Exception when calling Tickets#buyMuseumTickets: '
        . print_r($e->getResponseObject());
}

File: examples/museum/oseg_generated/python/BuyMuseumTicketsEventEntry.py
from datetime import date, datetime
from pprint import pprint

from openapimuseum_client import ApiClient, ApiException, Configuration, api, models

configuration = Configuration()

with ApiClient(configuration) as api_client:
    buy_museum_tickets = models.BuyMuseumTickets(
        ticket_type="event",
        ticket_date=date.fromisoformat("2023-09-05"),
        email="todd@example.com",
        ticket_id=None,
        event_id="dad4bce8-f5cb-4078-a211-995864315e39",
    )

    try:
        response = api.TicketsApi(api_client).buy_museum_tickets(
            buy_museum_tickets=buy_museum_tickets,
        )

        pprint(response)
    except ApiException as e:
        print("Exception when calling Tickets#buy_museum_tickets: %s\n" % e)
